src/serialize_sequence.cc

```diff
@@ -155,6 +155,9 @@ void store_step_setup_script(const std::filesystem::path& lua_file, const Sequen
     if (not stream.is_open())
         throw Error(gul14::cat("I/O error: unable to open file (", lua_file.string(), ")"));
 
+    if (not seq.get_maintainers().empty())
+        stream << "-- maintainers: " << seq.get_maintainers() << '\n';
```

Comment:
This reminds me: In `set_maintainers()`, we should make sure that there is no newline in the string. Otherwise we can corrupt the Lua file here.